com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1008,6 +1050,12 @@ public override NetcodeNetworkEvent PollEvent(out ulong clientId, out ArraySegme
             return NetcodeNetworkEvent.Nothing;
         }
 
+        /// <summary>
+        /// Send a payload to the specified clientId, data and channelName.
```

Comment:
Channel name is inaccurate. Only delivery types are used now.

Reply:
^

Reply:
True, but this was copied from the underlying Transport.cs, so that's probably where the fix should be made primarly.

Reply:
This is because the <inheritdoc> tag that is super useful is not supported basically anywhere. So copy paste has to be done.

Reply:
so, fix them in both places?